cartoframes/datasets.py

```diff
@@ -305,7 +311,7 @@ def _decode_geom(ewkb):
     return None
 
 
-def _clean_dataframe_from_carto(df, table_columns, decode_geom=False):
+def clean_dataframe_from_carto(df, table_columns, decode_geom=False):
```

Comment:
This is more of a `carto`-ify step instead of a cleaning step? Probably not important since this is largely an internal function, but something like `cartoify_dataframe` or `postprocess_dataframe` seems clearer.

Reply:
I didn't like the method name either 😅